src/rolodex/rolodexPrinter.js

```diff
@@ -1,11 +1,23 @@
 import people from './people.json';
 
+const{name, email, phone} = person;
+
 people.forEach(function (person) {
-  const names = person.name.split(' ');
-  const firstName = names[0];
-  const lastName = names[1];
-  const email = person.email;
-  const phone = person.phone;
+firstName = person.name;
+lastName = person.name;
+email = person.email;
+phone = person.phone;
 
-  console.log('First name: ' + firstName + '\nLast name: ' + lastName + '\nEmail: ' + email + '\nPhone number: ' + phone + '\n');
+  console.log(`First name:${firstName}\nLast name:${lastName}\nEmail:${email}\nPhone number:${phone}\n' `);
 });
+
+// people.forEach(function (person) {
+//   const names = person.name.split(' ');
+//   const firstName = names[0];
+//   const lastName = names[1];
+//   const email = person.email;
+//   const phone = person.phone;
+
+//   console.log('First name: ' + firstName + '\nLast name: ' + lastName + '\nEmail: ' + email + '\nPhone number: ' + phone + '\n');
+// });
```

Comment:
To get this working, you can just use this code + replace the `console.log` line with your correct template literal version. Then make the following changes:

1. Since `names` is an array, you can destructure the first/last names like this: `const [ firstName, lastName ] = person.name.split(' ');`
2. Since `email` and `phone` are properties being pulled out of the `person` object, they can be destructured as well, like this: `const { email, phone} = person;`.

Hopefully that helps.